vendor/modules.txt

```diff
@@ -275,7 +298,7 @@ google.golang.org/grpc/serviceconfig
 google.golang.org/grpc/stats
 google.golang.org/grpc/status
 google.golang.org/grpc/tap
-# google.golang.org/protobuf v1.25.0
+# google.golang.org/protobuf v1.26.0-rc.1
```

Comment:
@rajaSahil prometheus client library is enforcing above rc.1 dependency. Is it possible to enforce previous dependency v1.25.0 by adding a replace statement under go.mod?

Reply:
Done.